src/js/graphql_endpoint/modules/schema_to_graphql.ts

```diff
@@ -36,7 +37,7 @@ function propertyToGraphql(
     }
 
     if (property.primitive == "Int") {
-        prim = GraphQLInt;
+        prim = GraphQLFloat;
```

Comment:
GraphQLInt is only 32-bit and doesn't work with our timestamp ints

Reply:
that's insane -_- how sad